LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
this is a normal MIT right? asking since you wrote in the description that you were using BSD

Reply:
Yeah just regular MIT, must have accidentally put BSD in composer.json… I’m okay with whatever you want since your work is in here too.

Reply:
MIT is better imho, simpler and more widespread